scripts/README.md

```diff
@@ -91,23 +104,57 @@ api_token = "your_api_key_here"
 
 ```
 
-### Usage 
-Run the script after ensuring your CKAN application is running locally at `http://localhost`. The script will download CSV files.
+## CKAN & Ontario Theme
+
+Bash script: `setup_ckan_local.sh`
+
+This bash script installs CKAN and Ontario Theme locally, preferably on a Virutal Machine. The script follow the steps from the [official installation guide for installing CKAN from source](https://docs.ckan.org/en/2.9/maintaining/installing/install-from-source.htmlhttps:/).
+
+### Developer and Production Modes
+
+CKAN can be installed in Developer Mode, where the CKAN server is started manually through the commandline (`$ ckan -c /etc/ckan/default/ckan.ini run `) and is available at `localhost:5000` in the browser; Or CKAN can be installed in Production Mode where `supervisor` application is responsible for running the CKAN application and `nginx` is used as the webserver available directly at `localhost` in the browser. The Production Mode is how CKAN runs in production environment.
+
+The script installs CKAN+Ontario Theme in Production Mode.
+
+### What does the Script need?
+
+The script needs Solr, postgreSQL and relevant databases to be setup. These steps are recorded in the instructions below.
+
+For configuration, files in `config/` directory are used.
+
+* `config/ckan/ckan.ini` file is copied to`/etc/ckan/default/ckan.ini`, and modified during the course of execution of the script
+* `config/gtm/gtm.html` and `config/gtm/gtm_ns.html` are copied to `/usr/lib/ckan/default/src/ckanext-ontario_theme/ckanext/ontario_theme/templates/internal/`
+* `config/nginx/local_ckan_ssl`is copied to `/etc/nginx/sites-available/`
+* `config/supervisor/ckan-uwsgi.conf` and `config/supervisor/ckan-worker.conf`is copied to `/etc/supervisor/conf.d/`
+
+### Running the script
 
-1. Open a terminal and navigate to the repository directory.
-2. Execute the script with the following command:
+**Command line:**
+
+1. Do the steps common to the scripts in [common steps](#common-steps)
+2. setup solr and postgresql
+
+```bash
+sudo bash setup_solr.sh && sudo bash setup_postgres_local.sh
```

Comment:
running setup_solr.sh will give an error, I would suggest consolidating the solr installation above with this "running the script" section

Reply:
this line now refers to "Solr Installation" section, to properly install Solr8. Thanks for pointing out.